lib/npm.js

```diff
@@ -372,13 +370,12 @@ class Npm extends EventEmitter {
     return this.config.get('logs-dir') || join(this.cache, '_logs')
   }
 
-  get timingFile () {
-    return this.#timers.file
+  logPath = (file = '') => {
+    return resolve(this.logsDir, `${this.#runId}-${file}`)
```

Comment:
It looks like we're accessing this as a property not a function. Should this be a getter?  I don't see that we ever actually pass in a `file` param (cause we only ever access this as an attribute).

I may have missed something, admittedly.

Reply:
We pass it in as `path` to the timers and log files classes during `load()`. This enables those classes to append whatever they want to the path, but it should only ever be a file name suffix + extension.

Reply:
ooooh we are intentionally passing in the FUNCTION not a value.  That's ... a little confusing, and don't we lose the `this` context to npm when we do this?

Reply:
Since the only thing this function is doing is concatenating two strings, I really feel kind of strongly for the sake of readability that the log file code should do that: concatenate a given path with the extra metadata it wants to add.

Reply:
The fact that it doesn't lose `this` is due to it being a class property (eg `logPath = (file => {}` instead of `logPath() {}`) which I think makes your case even stronger. I'll make it a getter that returns a string.